sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -589,6 +610,11 @@ pi_result piPlatformsGet(pi_uint32 NumEntries, pi_platform *Platforms,
   static std::vector<pi_platform> PiPlatformsCache;
   static std::mutex PiPlatformsCacheMutex;
 
+  if (ZeValidationLayer) {
```

Comment:
Please add a comment that having these set in the environment **before** running zeInit will enable validation layer in Level-Zero loader.